json/Card/hida-tsuru.json

```diff
@@ -0,0 +1,31 @@
+[
+    {
+        "clan": "crab",
+        "cost": 4,
+        "deck_limit": 3,
+        "element": null,
+        "fate": null,
+        "glory": 1,
+        "honor": null,
+        "id": "hida-tsuru",
+        "influence_cost": null,
+        "influence_pool": null,
+        "military": "4",
+        "military_bonus": null,
+        "name": "Hida Tsuru",
+        "name_extra": null,
+        "political": "3",
+        "political_bonus": null,
+        "role_restriction": null,
+        "side": "dynasty",
+        "strength": null,
+        "strength_bonus": null,
+        "text": "<b>Reaction:</b> After a character is moved to or played into a conflict in which this character is participating - until the end of the conflict, this character gets +1[conflict-military] and +1[conflict-political].  (Unlimited.)",
```

Comment:
At the end of the sentence, there are two whitespace characters where only one is supposed to be, right before `(Unlimited)`

Reply:
I might be showing my age here - I have always put double spaces after full stops as I was taught at school :).  It does look as though the templating on the cards is only 1 space though and of course if that is the standard format on fiveringsdb I will change to a single space.